### PR TITLE
Chat: Copy conversation link to clipboard

### PR DESCRIPTION
**Pull request by PopDaph**
Adding a CopyToClipboard icon next to the Chat title, to copy the convo url.
The cursor transforms into a pointer when we hover the title and the icon. 
The icon changes to "check" to confirm it was done

Capture d’écran 2023-08-02 à 19 11 22

### CODE REVIEW

File: front/pages/w/[wId]/u/chat/[cId]/index.tsx [hunk 1]
```diff
@@ -1,8 +1,6 @@
 import {
   Button,
   ChatBubbleBottomCenterTextIcon,
-  CheckCircleIcon,
```
Comment (PopDaph) [inline front/pages/w/[wId]/u/chat/[cId]/index.tsx:4]: Removing unused imports, not related to the PR.

File: front/components/CopyToClipboard.tsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+export function CopyLinkToClipboard({
+  link,
+  children,
+}: React.PropsWithChildren<{ link: string }>) {
+  const [confirmed, setConfirmed] = useState<boolean>(false);
+  const handleClick = async () => {
+    await navigator.clipboard.writeText(link);
+    setConfirmed(true);
+    void setTimeout(() => {
```
Comment (fontanierh) [inline front/components/CopyToClipboard.tsx:13]: is the `void` needed here ?

**Review by fontanierh — DISMISSED** [review]
looks great !

File: front/components/CopyToClipboard.tsx (hunk 2 above)
Comment (PopDaph) [reply front/components/CopyToClipboard.tsx]: Nope, I removed!

**Comment by PopDaph** [comment]
I added a commit to remove the component as it was not really reusable because of the "truncate title logic", and set it directly to AppLayout.

**Pull request merged**